frontend/src/App.tsx

```diff
@@ -65,7 +78,7 @@ function App() {
 
         //
         default:
-          console.log(messageData)
+          console.log("unknown message type", messageData)
```

Comment:
is this needed?

Reply:
Its useful to see if the messages come through wrong